code/modules/research/xenobiology/crossbreeding/mutative.dm

```diff
@@ -0,0 +1,150 @@
+//All mutative extracts result in items so let's just put them in a single file for the ease of readability.
+/obj/item/slimecross/mutative
+	icon_state = "mutative"
+	name = "mutated"
+	effect = "mutative"
+
+/obj/item/slimecross/mutative/darkpurple
+	colour = "dark purple"
+	effect_desc = "Fully stocks up any pacman type generator with fuel."
+
+/obj/item/slimecross/mutative/darkblue
+	colour = "dark blue"
+	effect_desc = "Turns 50u of water into single stack of snow."
+
+/obj/item/slimecross/mutative/darkblue/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(!target.reagents)
+		return
+	var/datum/reagents/reggies = target.reagents
+	var/datum/reagent/water = reggies.has_reagent(/datum/reagent/water)
+	if(!water)
+		return
+	var/sheet_num = FLOOR(water.volume / 50,1)
+	if(!sheet_num)
+		return
+	playsound(src, 'sound/effects/bubbles.ogg', 50, TRUE)
+	reggies.remove_reagent(/datum/reagent/water,sheet_num*50)
+	var/locc = drop_location()
+	for(var/num in 1 to sheet_num)
+		new /obj/item/stack/sheet/mineral/snow(locc)
+
+/obj/item/slimecross/mutative/green
+	colour = "green"
+	effect_desc = "Single use core, use it in your hand to become immune to damage slow down and soft crit, but your blood becomes acid and if you die you are gibbed."
```

Comment:
yowza, this ones pretty strong

cut out the ignoring damage slowdown bit, and maybe dye the person who uses this a distinctive color, and this'll probably have enough downsides to make it NOT be a powergamer staple